.github/workflows/ecosystem-ci-trigger.yml

```diff
@@ -0,0 +1,92 @@
+name: ecosystem-ci trigger
+
+on:
+  issue_comment:
+    types: [created]
```

Comment:
I would add `workflow_dispatch:` too, so we can trigger it manually too

Reply:
In https://github.com/withastro/astro-ecosystem-ci the workflows are using `workflow_dispatch:`, so we can trigger the scheduled runs manually.

This action is only needed so we can use the comment commands to trigger, so we don't need to run this manually.